app/qml/map/MMMapController.qml

```diff
@@ -849,6 +849,18 @@ Item {
         linkText: qsTr( "See how to enable digitizing in your project." )
         link: __inputHelp.howToEnableDigitizingLink
       }
+
+      Connections {
+        target: __activeProject
+
+        function onProjectWillBeReloaded() {
+          recordingLayersModel.model.reset()
+        }
```

Comment:
This seems to call Qt's `reset` method - no need to do that, the code below will do the trick.
```suggestion

```